Shared/sdk/SharedUtil.File.hpp

```diff
@@ -29,22 +30,28 @@
     #include <sys/stat.h>
     #include <unistd.h>
     #include <limits.h>
-    #include <filesystem>
 #endif
 
 //
 // Returns true if the file exists
 //
 bool SharedUtil::FileExists(const SString& strFilename)
 {
-#ifdef WIN32
+#if _HAS_CXX17
+    namespace fs = std::filesystem;
+    return fs::is_regular_file(static_cast<const std::string&>(strFilename));
```

Comment:
Is static_cast really needed here?

Reply:
This cannot be slower because the string copying will occur in any way. By using `c_str` you will achieve the same behaviour as you already have. `static_cast` is something that you should not use if you can.